packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -27,15 +27,23 @@ import { Translation } from './Translation';
 
 const SPACING = 6;
 
-const Container = styled.div`
+const Container = styled.div<{ isClickable: boolean }>`
     display: flex;
+    justify-content: space-between;
     align-items: center;
     height: 28px;
     padding: 0 ${SPACING}px;
     background: ${({ theme }) => theme.BG_WHITE};
     border-bottom: 1px solid ${({ theme }) => theme.STROKE_LIGHT_GREY};
     font-size: ${variables.FONT_SIZE.TINY};
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    ${({ isClickable, theme }) =>
+        isClickable &&
+        `
+        cursor: pointer;
+        &:hover {
+            background: ${theme.BG_WHITE_ALT_HOVER};
```

Comment:
How about 

```
${ViewText} {
    text-decoration: underline;
    transform: translateX(-4px);
}
```

and `transition: transform 0.15s ease-in-out;` in `ViewText`?

Reply:
That looks pretty cool!